data/human/news.txt

```diff
@@ -1294,6 +1429,7 @@ news "human republic soldier"
 			"I hope I don't get posted to fighter duty next week. Those things are deathtraps. By the time you get close enough to fire your lasers, you've already lost half your shields!"
 			"Nobody thinks about the Alphas anymore, but don't forget that they just disappeared. They could return at any time. If that happens, I'm going to kill every last one of 'em."
 			"While it seems pointless to have the Bactrian locked behind a license, don't forget that it's one of the oldest designs in human space. It was the Navy Carrier of its time - and it's still comparable to one."
+			"We recently went to full alert, because somebody screamed `It's scrambled!`. Turns out some rookie in the galley was enthusiastic about making scrambled eggs."
```

Comment:
```suggestion
			"We recently went to full alert, because somebody screamed 'It's scrambled!'. Turns out some rookie in the galley was enthusiastic about making scrambled eggs."
```